src/commands/components/versions.ts

```diff
@@ -12,7 +12,10 @@ interface ComponentVersion {
 export default class ComponentVersions extends BaseCommand {
   static aliases = ['component:versions', 'component:version'];
   static description = 'Search component versions of a particular component';
-
+  static examples = [
+    'architect component:versions myComponentName',
```

Comment:
lets be consistent in naming `mycomponent` or `mycomponentname`

Reply:
renamed all example values {value}Name to {value} to be consistent